backend/fms_core/resources/sample.py

```diff
@@ -272,7 +296,11 @@ def import_field(self, field, obj, data, is_m2m=False):
             # Ignore importing this, since it's a computed property.
             return
 
-        super().import_field(field, obj, data, is_m2m)
+        try:
+            super().import_field(field, obj, data, is_m2m)
+        except Exception as e:
+            pass
```

Comment:
...or at least make sure you are more precise with your except to catch the case you want to ignore as precisely as possible and leave a comment in the code as to what condition this applies to so we can have the information without parsing the whole code for such an unorthodox behaviour...

Reply:
Yes, I looked into it in detail and there's a problem when there's no container_kind. Instead of raising a Blank Exception (which is what is expected) and continuing parsing the row it throws and exception and it stops parsing that row. One option would be: 

```suggestion
            raise ValidationError({"container_kind": ValidationError([f"This field cannot be blank."], code="invalid")})
```

I tested it and it works.

Reply:
The method you are in parse each field of the template (not only container_kind) if the import_field raise an error for another field this is going to get really annoying to track down...

I would suggest you do the catching of this error in the "field.attribute == "container_barcode" section raising the validation error if the container_kind is missing. 

Additionally, for the exception raised at the level of super.import_field, once the case is tested positively to make sure a validation error is raised if there is a problem you may be able to bypass the exception by setting self.fields_manually_excluded.extend(["container_kind"]) before raising your error...
(and again maybe not)...

This is hacky but at the moment its how we get by at other places (you can check ressource.extraction for an example of what is done elsewhere). If this does not work you'll have to dig a bit deeper ;).